### PR TITLE
Fix outdated helm values parameter - sparkJobNamespaces in docs

### PR DESCRIPTION
**Pull request by EricGao888**
### 🛑 Important:
Please open an issue to discuss significant work before you start. We appreciate your contributions and don't want your efforts to go to waste!

For guidelines on how to contribute, please review the [CONTRIBUTING.md](CONTRIBUTING.md) document.

## Purpose of this PR
Provide a clear and concise description of the changes. Explain the motivation behind these changes and link to relevant issues or discussions.

**Proposed changes:**
* The parameter `--set sparkJobNamespace=default` to set spark operator listen to a specific namespace for spark applications in `README` docs is outdated and inconsistent with that in `values.yaml`
https://github.com/kubeflow/spark-operator/blob/ccb3ceb54b6f09de8c67d96484fc911d122dcee3/charts/spark-operator-chart/values.yaml#L56

## Change Category
Indicate the type of change by marking the applicable boxes:

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that could affect existing functionality)
- [x] Documentation update

## Checklist
Before submitting your PR, please review the following:

- [x] I have conducted a self-review of my own code.
- [x] I have updated documentation accordingly.
- [x] I have added tests that prove my changes are effective or that my feature works.
- [x] Existing unit tests pass locally with my changes.

### CODE REVIEW

**Comment by yuchaoran2011** [comment]
Thanks @EricGao888 for the contribution. Please fix the DCO failure: https://github.com/kubeflow/spark-operator/pull/2002/checks?check_run_id=24187032939

**Comment by EricGao888** [comment]
> Thanks @EricGao888 for the contribution. Please fix the DCO failure: https://github.com/kubeflow/spark-operator/pull/2002/checks?check_run_id=24187032939

Hi @yuchaoran2011 , thx for your kindly reminder. I've fix the DCO failure : )

File: docs/quick-start-guide.md [hunk 1]
```diff
@@ -236,9 +236,9 @@ Refer to the Helm [documentation](https://helm.sh/docs/helm/helm_upgrade/) for m
...
+The Spark Job Namespace value defines the namespace(s) where `SparkApplications` can be deployed. The Helm chart value for the Spark Job Namespace is `sparkJobNamespaces`, and its default value is `[]`, as defined in the Helm chart's [README](../charts/spark-operator-chart/README.md). Note that in the [Kubernetes apimachinery](https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/apimachinery) project, the constants `NamespaceAll` and `NamespaceNone` are both defined as the empty string. In this case, the empty string represents `NamespaceAll`. When set to `""`, the Spark Operator supports deploying `SparkApplications` to all namespaces. The Helm chart will create a service account in the namespace where the spark-operator is deployed. In order to successfully deploy `SparkApplications`, you will need to ensure the driver pod's service account meets the criteria described in the [service accounts for driver pods](#about-the-service-account-for-driver-pods) section.
```
Comment (peter-mcclonski) [inline docs/quick-start-guide.md:239]: #2010 changes the default value from `[]` to `[""]` in order to fix behavior issues.

**Comment by EricGao888** [comment]
Close this PR due to the merge of #2000

Problem solved already.

**Pull request closed without merging**